FirebaseFirestoreInternal/FirebaseFirestore/FIRBSONBinaryData.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2025 Google LLC
```

Comment:
A note on these redirect headers– they were added to preserve existing header imports when we reorganized the Firestore modules so that the top-most `FirebaseFirestore` module was actually a Swift module. Unless these headers need to be imported in C++ clients, we probably could remove them in favor of customers importing them with the module style import of FirebaseFirestore.

Reply:
I'm not positive, but found [this](https://github.com/search?q=repo%3Afirebase%2Ffirebase-cpp-sdk%20firebasefirestoreinternal&type=code) in the cpp sdk.